src/xo-app/home/vm-item.js

```diff
@@ -173,17 +220,20 @@ export default class VmItem extends Component {
               {vm.docker ? <Icon icon='vm-docker' /> : null}
             </span>
           </Col>
-          <Col largeSize={3} className={styles.itemExpanded}>
-            {map(vm.addresses, address => <span key={address} className='tag tag-info tag-ip'>{address}</span>)}
-          </Col>
-          <Col mediumSize={3} className='hidden-sm-down'>
+          <Col mediumSize={2} className='hidden-sm-down'>
             {resourceSet && <span>{_('homeResourceSet', { resourceSet: <Link to={`self?resourceSet=${resourceSet.id}`}>{resourceSet.name}</Link> })}</span>}
           </Col>
-          <Col mediumSize={3}>
+          <Col mediumSize={6} className={styles.itemExpanded}>
+            {map(vm.addresses, address => <span key={address} className='tag tag-info tag-ip'>{address}</span>)}
+          </Col>
+          <Col mediumSize={6}>
             <span style={{fontSize: '1.4em'}}>
               <HomeTags type='VM' labels={vm.tags} onDelete={this._removeTag} onAdd={this._addTag} />
             </span>
           </Col>
+          <Col mediumSize={6} className={styles.itemExpanded}>
+            {this._isRunning && <MiniStats vmId={this.props.item} />}
```

Comment:
Does this work well? You're having a weird `Row > Col > Row > Col` tree without another `Container`.

Reply:
I don't see any functional issue. If you think it's better to add a container, I can.

Reply:
I think `Container` should only be used at the root level.

Reply:
You should be able to use any node you want inside a `Row`, so why not another `Container`?

Reply:
Because a `Container` is meant to be used at the root level, for instance a non-fluid container as a defined width.